tests/integration/tox.ini

```diff
@@ -1,47 +1,46 @@
 [tox]
-no_package = True
```

Comment:
All these changes are to make this more compatible with tox 3.  Why tox 3? Because that's what you get with apt on focal and jammy. Why not pip install it?  Arm64 runners were having severe trouble loading python libraries from a virtual env on the GH runners for reasons i never was able to understand.  I think it's a combination of running tox with sg but i'm not sure.  LD_PYTHON_PATH got lost though -- using the system-wide `tox` improves this